triton-server/src/contact-service.ts

```diff
@@ -83,7 +83,8 @@ export function start() {
                             await send(
                                 `${subject}`,
                                 `${subject}.<br/>
-                        The dataset can be downloaded using ${request.type} using the credential provided to you.`,
+                        The dataset can be downloaded using ${request.type} using the credential provided to you.<br/>
+                        If you forgot your credential, you can reset your password in the data portal.`,
```

Comment:
Isn't that part of my other PR?
https://github.com/c3g/triton/pull/64/files#diff-fe59c7bf81553ba3edd7f2ee51fd760eebed51ec5969cfd82ab32f5ae5e68ba7

Reply:
You may have forgotten to rebase.

Reply:
i rebased main unto my branch